.changelog/adfe779a-b5b4-44bd-8e23-5cdb4db5aef8.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "adfe779a-b5b4-44bd-8e23-5cdb4db5aef8",
+  "type": "bugfix",
+  "collapse": true,
```

We only need to collapse for sweeping changes that touched many modules.

```suggestion
  "collapse": false,
```